SimDataFormats/Associations/interface/VertexToTrackingVertexAssociator.h

```diff
@@ -27,17 +29,13 @@ namespace reco {
                                                        const edm::Handle<TrackingVertexCollection>& tVCH ) const {
       return m_impl->associateSimToReco(vCH,tVCH);
     }
-
-    void swap(VertexToTrackingVertexAssociator& iOther) {
-      std::swap(m_impl, iOther.m_impl);
-    }
   private:
     VertexToTrackingVertexAssociator(const VertexToTrackingVertexAssociator&) = delete; // stop default
     
     const VertexToTrackingVertexAssociator& operator=(const VertexToTrackingVertexAssociator&) = delete; // stop default
     
     // ---------- member data --------------------------------
-    VertexToTrackingVertexAssociatorBaseImpl* m_impl;
+    std::unique_ptr<VertexToTrackingVertexAssociatorBaseImpl> m_impl;
```

Comment:
Schema change

Reply:
This is transient as well
https://github.com/cms-sw/cmssw/blob/38a95c4d8839d94350b8a27837f7fac947e1909e/SimDataFormats/Associations/src/classes_def.xml#L9-L10